webserver/views/data.py

```diff
@@ -93,7 +93,7 @@ def summary(mbid):
     except db.exceptions.NoDataFoundException:
         summary_data = {}
 
-    if "highlevel" in summary_data:
+    if "highlevel" in summary_data and summary_data["highlevel"]:
```

Comment:
what can `summary_data["highlevel"]` be which is bad?
How about `if summary_data.get("highlevel", {})`?